spec/factories/events.rb

```diff
@@ -8,4 +8,9 @@
     prefecture_id { [*1..47].sample }
     user
   end
+
+  trait :woman_only_event do
+    only_woman { true }
+    user { FactoryBot.create(:user, :woman_user) }
```

Comment:
```suggestion
    user { create(:user, :woman_user) }
```
でいけませんでしたっけ? (いけなかったら申し訳ない)

Reply:
743387b49cc59c7b07dc8ef3ac34a04edb59309f で修正しました。

例 ↑ 上記のように、指摘に対して1コミットとするといいですね